synapse_data_warehouse/synapse_raw/tables/V2.23.0__add_version_history_to_nodesnapshot.sql

```diff
@@ -0,0 +1,9 @@
+-- Configure environment
+USE ROLE sysadmin;
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP,CP02
+USE WAREHOUSE compute_xsmall;
+
+-- Add `version_history` column
+ALTER TABLE nodesnapshots
+  ADD COLUMN version_history VARIANT
+  COMMENT 'The list of entity versions, at the time of the snapshot.';
```

Comment:
nit (feel free to ignore): The list of all _previous_ entity versions

Reply:
Actually, I don't think that's accurate. I see a `version_history` value that appears to include the current latest version. It looks like this:

```
[
  {
    "id": "syn9606778",
    "isLatestVersion": true,
    "modifiedBy": "1",
    "modifiedByPrincipalId": "1",
    "modifiedOn": 1731441964134,
    "versionComment": "in progress",
    "versionLabel": "in progress",
    "versionNumber": 2
  },
  {
    "id": "syn9606778",
    "isLatestVersion": false,
    "modifiedBy": "1",
    "modifiedByPrincipalId": "1",
    "modifiedOn": 1731441964101,
    "versionLabel": "1",
    "versionNumber": 1
  }
]
```

Reply:
oh I see - thanks for checking!